lib/genericCommand.js

```diff
@@ -115,11 +115,10 @@ module.exports = function(cmd) {
     if (typeof postCmd === 'function') {
       // Apply a useful scope in these functions
       var post = function(params, cb) {
-        postCmd.apply(cmd, [params, cb]);
+        postCmd.apply(cmd, [argv, params, cb]);
```

Comment:
@trepel following the local test with it and this PR. Did you update the PR for your tests? 


```
Camilas-MBP:fh-fhc cmacedo$ git checkout FH-3657
Already on 'FH-3657'
Camilas-MBP:fh-fhc cmacedo$ ./bin/fhc.js app stage --app=xdhzjpmzhhzab6ap755pju3y --env=dev --runtime=node4
Progress: 5%
[[
  {
    "action": {},
    "cacheKey": "d43868c71923e3df2a4a6b28c1e30ba0",
    "error": "",
    "log": [
      "Performing pre-deploy checks",
      "Pre-deploy checks passed",
      "Deploy operation scheduled",
      "Checking app runtime\n Runtime is ok. Creating app\n [stdout] Package.json unchanged, not running npm.\n Updating app state\n Starting app\n App deployed\n "
    ],
    "progress": 5,
    "status": "complete"
  }
]]
Camilas-MBP:fh-fhc cmacedo$ 

```